### PR TITLE
Update room info

### LINKED ISSUE #363: Update room details to match new design
[Zeplin](https://app.zeplin.io/project/5a1308cdae8ae89cae5c15af/screen/5a9ec788ac81437516404876)

### PR DESCRIPTION
**Pull request by tbergq**
closes #363

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Haha, it looks so good on the design but it's so bad in reality:

screen shot 2018-03-14 at 10 30 26

Brilliant... :)

**Comment by mrtnzlml** [comment]
@tbergq Could you please improve this elements align? 
screen shot 2018-03-14 at 11 58 18

**Comment by tbergq** [comment]
Should be dooable

File: app/hotels/src/singleHotel/roomList/RoomBadges.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        style={styles.badgeContainer}
+        textStyle={styles.badgeText}
+      />,
+    );
+  }
+
+  if (isRefundable) {
+    badges.push(
+      <AdaptableBadge
+        icon={<TextIcon style={styles.iconStyle}>'</TextIcon>} // eslint-disable-line react/no-unescaped-entities
```
Comment (mrtnzlml) [inline app/hotels/src/singleHotel/roomList/RoomBadges.js]:
```diff
- icon={<TextIcon style={styles.iconStyle}>'</TextIcon>} // eslint-disable-line react/no-unescaped-entities
+ icon={<TextIcon style={styles.iconStyle}>{"'"}</TextIcon>}
```

**Comment by tbergq** [comment]
@mrtnzlml Alignment improved

**Pull request merged**